app.R

```diff
@@ -1,49 +1,103 @@
 
-    # Sidebar with a slider input for number of bins
+    titlePanel("Average DBH for Each Genus in the SCBI Census"),
+
+
     sidebarLayout(
-        sidebarPanel(
-            sliderInput("bins",
-                        "Number of bins:",
-                        min = 1,
-                        max = 50,
-                        value = 30)
-        ),
-
-        # Show a plot of the generated distribution
-        mainPanel(
-           plotOutput("distPlot")
-        )
+      sidebarPanel(
+        selectInput("census_year", "Census Year",
+                    choices = c("2008" = "1","2013" = "2", "2018" = "3"),
+                    selected = "3"),
+        checkboxGroupInput("Genus", "Genus",
+                           choices = c("Acer","Ailanthus","Amelanchier","Asimina",
+                                       "Berberis","Carpinus","Carya","Castanea","Celtis",
+                                       "Cercis","Chionanthus","Cornus","Corylus",
+                                       "Crataegus","Diospyros","Elaeagnus","Euonymus",
+                                       "Fagus","Fraxinus","Hamamelis","Ilex","Juglans",
+                                       "Juniperus","Lindera","Liriodendron","Lonicera","Nyssa",
+                                       "Paulownia","Pinus","Platanus","Prunus","Quercus",
+                                       "Rhododendron","Robinia","Rosa","Rubus","Sambucus",
+                                       "Sassafras","Tilia","Ulmus","Viburnum"),
+                           selected = c("Acer","Ailanthus","Amelanchier","Asimina",
+                                        "Berberis","Carpinus","Carya","Castanea","Celtis",
+                                        "Cercis","Chionanthus","Cornus","Corylus",
+                                        "Crataegus","Diospyros","Elaeagnus","Euonymus",
+                                        "Fagus","Fraxinus","Hamamelis","Ilex","Juglans",
+                                        "Juniperus","Lindera","Liriodendron","Lonicera","Nyssa",
+                                        "Paulownia","Pinus","Platanus","Prunus","Quercus",
+                                        "Rhododendron","Robinia","Rosa","Rubus","Sambucus",
+                                        "Sassafras","Tilia","Ulmus","Viburnum"),
+                           inline = FALSE),
+      ),
+      plotOutput("avg_dbh")
     )
 )
 
-# Define server logic required to draw a histogram
+
+
+
 server <- function(input, output) {
 
-    output$distPlot <- renderPlot({
-        # generate bins based on input$bins from ui.R
-        x    <- faithful[, 2]
-        bins <- seq(min(x), max(x), length.out = input$bins + 1)
+    output$distplot <- renderPlot({
 
-        # draw the histogram with the specified number of bins
-        hist(x, breaks = bins, col = 'darkgray', border = 'white')
+        ggplot(data = genus_dbh, mapping = aes(x = Genus, y = avg_dbh, fill = census_year))+
```

Comment:
Looking at this plot however, it seems what you input in your input panels (Genus and Census Year) doesn't get linked at all to the plot. In other words, there will be nothing interactive about the plot as it's written. Check out the lecture on Shiny Apps to see how to link the two